gate-web/src/main/groovy/com/netflix/spinnaker/gate/filters/CorsFilter.groovy

```diff
@@ -43,6 +45,8 @@ class CorsFilter implements Filter {
     String origin = request.getHeader("Origin")
     if (!originValidator.isValidOrigin(origin)) {
       origin = '*'
+    } else if (!originValidator.isExpectedOrigin(origin)) {
+      log.debug("Origin header {} is not in expected origin whitelist.", origin)
```

Comment:
thoughts on also logging the http method?

Reply:
it should be included in the origin